src/components/LegalityPageContent/LegalityPageContent.tsx

```diff
@@ -0,0 +1,197 @@
+
+const TRADETRUST_LEGAL_ARTICLE_TITLE = "Tradetrust Legal Analysis Article";
+
+const Card = ({ title, src }: { title: string; src: string }) => {
+  return (
+    <div className="h-full rounded-lg bg-cerulean-500 text-white p-4 text-center">
+      <img className="mx-auto mb-4" src={src} alt={title} />
+      <h5>{title}</h5>
+    </div>
+  );
+};
+
+const Section = ({
+  title,
+  img,
+  children,
+  className,
+}: {
+  title: string;
+  img: {
+    src: string;
+    alt: string;
+  };
+  children: React.ReactElement;
+  className?: string;
+}) => {
+  return (
+    <section className={`py-12 lg:py-16 ${className}`}>
+      <div className="container">
+        <h2 className="text-center leading-none">{title}</h2>
+        <div className="flex flex-wrap items-center py-8">
+          <div className="w-full lg:w-2/5">
+            <div className="max-w-[480px] mx-auto lg:ml-0">
+              <img className="mb-8 lg:mb-0" src={img.src} alt={img.alt} />
+            </div>
+          </div>
+          <div className="w-full lg:w-3/5">
+            <div className="px-0 lg:px-12">{children}</div>
+          </div>
+        </div>
+      </div>
+    </section>
+  );
+};
+
+const SectionGuide = () => {
+  const downloadArticle = () => {
+    gaEvent({
+      category: GaCategory.FILE_DOWNLOAD,
+      action: GaAction.LEGAL_ARTICLE_DOWNLOAD,
+      label: TRADETRUST_LEGAL_ARTICLE_TITLE,
+    });
+  };
+
+  return (
+    <section className="py-12 lg:py-16 bg-white">
+      <div className="container">
+        <h2 className="text-center leading-none">TradeTrust Legality Guide</h2>
+        <div className="flex flex-wrap py-8 justify-center -mx-8">
+          <div className="w-full lg:w-[320px] px-8 mb-4 lg:mb-0">
+            <Card
+              title="How TradeTrust tackles the twin problems of legal and technical interoperability that have hindered the wider adoption of eBLs in global trade"
+              src="/static/images/legality/icon-cost.svg"
+            />
+          </div>
+          <div className="w-full lg:w-[320px] px-8 mb-4 lg:mb-0">
+            <Card
+              title="A legal analysis of TT eBLs under the MLETR, under Singapore law, under English law and under US (New York and Delaware) law"
+              src="/static/images/legality/icon-legal.svg"
+            />
+          </div>
+          <div className="w-full lg:w-[320px] px-8 mb-4 lg:mb-0">
+            <Card
+              title="Practical advice to the Market when considering the implementation and use of TT eBLs in trade scenarios involving MLETR and non-MLETR aligned jurisdictions"
+              src="/static/images/legality/icon-location.svg"
+            />
+          </div>
+        </div>
+        <div className="w-full text-center my-8">
+          <a
+            download={TRADETRUST_LEGAL_ARTICLE_TITLE}
+            href="/static/images/legality/tradetrust-legal-article.pdf"
+            onClick={downloadArticle}
```

Comment:
Pretty sure the [click](https://youtu.be/X-KwYX2u8e4?si=3WKC1F3S68XT7dI5) count for this will be very high!😄